lib/gamelogic.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/LineLength
+
+class Game
+  attr_accessor :players
+  def initialize
+    @players = []
+  end
+
+  def toset_player(player)
+    @players << player
+  end
+end
+
+class Board
+  attr_accessor :counter
+  attr_reader :result, :contador, :winner
+
+  def initialize
+    @board = { a1: '', a2: '', a3: '', b1: '', b2: '', b3: '', c1: '', c2: '', c3: '' }
+    @turn = 'X'
+    @result = ''
+    @contador = 0
+    @winner = ''
+    @counter = 1
+  end
+
+  def mark_choice(choice, _counter)
+    @contador += 1
+    @turn = @counter == 1 ? 'X' : 'O'
+    @board[choice.to_sym] = 'X' if @turn == 'X'
+    @board[choice.to_sym] = 'O' if @turn == 'O'
+    check_victory
+  end
+
+  def display_board
+    puts "--------------------------\n\s\s\s\s a|\s #{@board[:a1]}\s |\s #{@board[:a2]}\s |\s #{@board[:a3]}\s |\n\s\s\s\s b|\s #{@board[:b1]}\s |\s #{@board[:b2]}\s |\s #{@board[:b3]}\s |\n\s\s\s\s c|\s #{@board[:c1]}\s |\s #{@board[:c2]}\s |\s #{@board[:c3]}\s | \n--------------------------"
+  end
```

Comment:
* [ ] No `puts/print` and `get.chomp` outside `main.rb` file.
  - All code responsible for UI interactions should go inside `main.rb`, sorry for not mentioning this in previous reviews, but it's a requirement.
  - These methods can go inside `main.rb` after doing the necessary changes :nerd_face: